core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -273,44 +275,56 @@ var StoryEditorPage = function() {
   };
```

Comment:
Can we similarly wait for `storyTitleField` to be visible before getting its attribute in `this.expectTitleToBe` above?